src/main/java/sorald/sonar/SonarStaticAnalyzer.java

```diff
@@ -15,22 +15,15 @@
 import sorald.rule.StaticAnalyzer;
 
 public class SonarStaticAnalyzer implements StaticAnalyzer {
-    private final File projectRoot;
-    private final SonarLintEngine sonarLint;
-
-    public SonarStaticAnalyzer(File projectRoot) {
-        this.projectRoot = projectRoot;
-        this.sonarLint = SonarLintEngine.getInstance();
```

Comment:
I did this because I wanted to "instantiate" the instance of `SonarLintEngine` only and then I store it in the field here. This was done for faster execution of test cases. I now realise I have already done that by making `theOnlyInstance` static [here](https://github.com/SpoonLabs/sorald/blob/master/src/main/java/sorald/sonar/SonarLintEngine.java#L62). Thanks for the refactoring.